src/t8_geometry/t8_geometry_implementations/t8_geometry_examples.cxx

```diff
@@ -291,6 +291,66 @@ t8_geometry_triangulated_spherical_surface::t8_geom_evaluate (t8_cmesh_t cmesh,
   }
 }
 
+/**
+ * Map the faces of a unit cube to a spherical surface.
+ * \param [in]  cmesh      The cmesh in which the point lies.
+ * \param [in]  gtreeid    The global tree (of the cmesh) in which the reference point is.
+ * \param [in]  ref_coords  Array of \a dimension many entries, specifying a point in [0,1]^dimension.
+ * \param [out] out_coords  The mapped coordinates in physical space of \a ref_coords.
+ */
+void
+t8_geometry_quadrangulated_spherical_surface::t8_geom_evaluate (t8_cmesh_t cmesh, t8_gloidx_t gtreeid,
+                                                                const double *ref_coords, const size_t num_coords,
+                                                                double *out_coords) const
+{
+  double n[3]; /* Normal vector. */
+  double r[3]; /* Radial vector. */
+  double p[3]; /* Vector on the plane. */
+
+  const t8_locidx_t ltreeid = t8_cmesh_get_local_id (cmesh, gtreeid);
+  const double *tree_vertices = t8_cmesh_get_tree_vertices (cmesh, ltreeid);
+
+  t8_geom_linear_interpolation (t8_element_centroid_ref_coords[T8_ECLASS_QUAD], tree_vertices, 3, 2, n);
```

Comment:
```suggestion
  t8_geom_linear_interpolation (t8_element_centroid_ref_coords[T8_ECLASS_QUAD], active_tree_vertices, 3, 2, n);
```
Just replace `tree_vertices` with `active_tree_vertices` and it should work just fine

Reply:
Done!